packages/@o3r/core/migration.json

```diff
@@ -5,6 +5,11 @@
       "version": "8.2.0-rc.3",
       "description": "Updates of the Otter Library to v8.2.*",
       "factory": "./schematics/ng-update/index#updateV8_2"
+    },
+    "migration-v10_0": {
+      "version": "10.0.0",
```

Comment:
"version": "10.0.0-next.0",

Reply:
We already have `10.0.0-alpha.0` `10.0.0-rc.0` 
we should decide which one we want to put in place 
issue with `10.0.0-next.0` is that migrating to this version will not include this script has it will be deploy at least on `10.0.0-next.22`